lib/structuraid_core/engineering/locations/coordinates_system.rb

```diff
@@ -0,0 +1,59 @@
+module StructuraidCore
+  module Engineering
+    module Locations
+      class CoordinatesSystem < Base
+        attr_reader :relative_locations
+
+        def initialize(anchor_location:)
+          @anchor_location = anchor_location
+          @theta = 0.0
```

Comment:
Does `@theta` really need to be an instance variable?

Reply:
yeah, I think so, because what if we need to roll back the axes' rotation?

Reply:
solving...

Reply:
done! think